db/migrate/20221018225521_add_post_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostToComments < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :comments, :post, null: false, foreign_key: true
+    end
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly remove this migration file as the project requirements only stated that you create tables for User, Post, Comment, and like only.